### PR TITLE
added timeout for wait_db_up, created a yaml for the test

### PR DESCRIPTION
**Pull request by harelassayag**

### CODE REVIEW

File: build_cluster_test.py [hunk 1]
```diff
@@ -82,7 +82,7 @@ def test_use_public_dns_names(self):
         for node in self.db_cluster.nodes:
             node.remoter.run('sudo systemctl start scylla-server.service')
             node.remoter.run('sudo systemctl start scylla-jmx.service')
-            node.wait_db_up()
+            node.wait_db_up(timeout=300)
             node.wait_jmx_up()
 
         base_cmd_w = ("cassandra-stress write no-warmup cl=QUORUM duration=5m "
```
Comment (larisau) [inline build_cluster_test.py]: probably it's better to define stress command in the yaml, if not - remove the unused definitions from the yaml

File: tests/build-cluster-dns.yaml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+             "cassandra-stress counter_write cl=QUORUM duration=5760m -schema 'replication(factor=3) compaction(strategy=DateTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=1000 -pop se
+q=1..1000000",
+             "cassandra-stress user profile=/tmp/cs_mv_profile.yaml ops'(insert=3,read1=1,read2=1,read3=1)' cl=QUORUM duration=5760m -port jmx=6868 -mode cql3 native -rate threads=100"]
+stress_read_cmd: ["cassandra-stress read duration=5760m -port jmx=6868",
+                  "cassandra-stress counter_read duration=5760m -port jmx=6868"]
+n_db_nodes: 3
+n_loaders: 2
+n_monitor_nodes: 1
+nemesis_class_name: 'NoOpMonkey'
+nemesis_interval: 5
```
Comment (larisau) [inline tests/build-cluster-dns.yaml]: if it is not used(e.g. nemesis), remove it from yaml

File: build_cluster_test.py [hunk 3]
```diff
@@ -82,7 +82,7 @@ def test_use_public_dns_names(self):
         for node in self.db_cluster.nodes:
             node.remoter.run('sudo systemctl start scylla-server.service')
             node.remoter.run('sudo systemctl start scylla-jmx.service')
-            node.wait_db_up()
+            node.wait_db_up(timeout=300)
             node.wait_jmx_up()
```
Comment (roydahan) [inline build_cluster_test.py]: Add a verify stress_thread after the stress.

Comment (roydahan) [inline build_cluster_test.py]: also, change the loader c-s command to append the DNS name after "-node" instead of the IP (if possible).

**Comment by harelassayag** [comment]
created a new PR
https://github.com/scylladb/scylla-cluster-tests/pull/432

**Pull request closed without merging**